app/views/nodes/index.html.slim

```diff
@@ -48,15 +47,13 @@ h1 Nodes
           td.text-center
             - if m.role == "master"
               = radio_button_tag "roles[master]", m.id, true, disabled: true
-            - elsif Minion.all.collect(&:role).include?("master")
+            - elsif Minion.exists?(role: "master")
               = radio_button_tag "roles[master]", m.id, false, disabled: true
             - else
               = radio_button_tag "roles[master]", m.id
   .clearfix.text-right.steps-container
     a.btn.btn-danger[href="/"] Back
-    - if Minion.all.collect(&:role).include?("master")
-      = submit_tag "Apply", class: "btn btn-primary", disabled: true
+    - if Minion.exists?(role: "master")
+      = submit_tag "Bootstrap Cluster", class: "btn btn-primary", disabled: true
```

Comment:
Clicking on this button when no minions have registered (and thus no master selected) results to:

```

ActionController::ParameterMissing in NodesController#update_nodes
param is missing or the value is empty: roles
```